bin/build.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path')
+const browserify = require('browserify')
+const resolve = require('resolve')
+const xtend = require('xtend')
+const fs = require('fs')
+const mkdirp = require('mkdirp')
+
+const defaults = {
+  optimize: false,
+  entry: '.',
+  html: {},
+  css: {},
+  js: {}
+}
+
+const cwd = process.cwd()
+
+// resolve a path according to require.resolve algorithm
+// string -> string
+function resolveEntryFile (relativePath) {
+  const first = relativePath.charAt(0)
+  const entry = ['.', '/'].includes(first) ? relativePath : './' + relativePath
+  return resolve.sync(entry, {basedir: cwd})
+}
+
+function build (options, cb) {
+  const bankai = require('../')({
```

Comment:
could we move this import statement to the top of the file? - inlining imports like this can be surprising